package/base-files/files/sbin/sysupgrade

```diff
@@ -213,10 +213,14 @@ build_list_of_backup_overlay_files() {
 	( cd /overlay/upper/; find .$SAVE_OVERLAY_PATH \( -type f -o -type l \) $find_filter | sed \
 		-e 's,^\.,,' \
 		-e '\,^/etc/board.json$,d' \
+		-e '\,^/etc/luci-uploads/.placeholder,d' \
 		-e '\,/[^/]*-opkg$,d' \
 		-e '\,^/etc/urandom.seed$,d' \
+		-e '\,^/etc/apk/world$,d' \
 		-e "\,^$INSTALLED_PACKAGES$,d" \
 		-e '\,^/usr/lib/opkg/.*,d' \
+		-e '\,^/lib/apk/.*,d' \
+		-e '\,^/run,d' \
```

Comment:
isn't this linked to /tmp ?

Reply:
Yes, but it looks like it is created by /etc/init.d/boot:

boot() {
...
        mkdir -p /var/run
        ln -s /var/run /run
        ln -s /var/lock /run/lock


So, it is missing in the /rom. A full backup will include that file. Maybe we should incorporate the /run into the squashfs image (but it will be a broken link until /var/run is created).